react-app/src/components/property/PropertyTable.tsx

```diff
@@ -0,0 +1,289 @@
+import React, { MutableRefObject } from 'react';
+import { CustomMenuItem, FilterSearchDataGrid } from '../table/DataTable';
+import { Box, SxProps, Tooltip, Typography, lighten, useTheme } from '@mui/material';
+import { GridApiCommunity } from '@mui/x-data-grid/internals';
+import { Check } from '@mui/icons-material';
+import { GridColDef, GridColumnHeaderTitle } from '@mui/x-data-grid';
+import { dateFormatter } from '@/utils/formatters';
+import ClassificationIcon from './ClassificationIcon';
+
+interface IClassificationToolTipItem {
+  color: string;
+  backgroundColor: string;
+  title: string;
+}
+
+const ClassificationToolTipItem = (props: IClassificationToolTipItem) => {
+  return (
+    <Box display={'flex'} flexDirection={'row'} alignItems={'center'} gap={'12px'}>
+      <Box
+        display={'flex'}
+        alignItems={'center'}
+        justifyContent={'center'}
+        sx={{
+          width: '16px',
+          height: '16px',
+          borderRadius: '50%',
+          backgroundColor: props.backgroundColor,
+        }}
+      >
+        <Box
+          sx={{
+            width: '8px',
+            height: '8px',
+            borderRadius: '50%',
+            opacity: '100%',
+            backgroundColor: props.color,
+          }}
+        ></Box>
+      </Box>
+      <Typography fontSize={'0.8rem'}>{props.title}</Typography>
+    </Box>
+  );
+};
+
+const PropertyTable = () => {
+  const theme = useTheme();
+
+  const classificationColorMap = {
+    0: { textColor: theme.palette.blue.main, bgColor: theme.palette.blue.light },
+    1: { textColor: theme.palette.success.main, bgColor: theme.palette.success.light },
+    2: { textColor: theme.palette.info.main, bgColor: theme.palette.info.light },
+    3: { textColor: theme.palette.info.main, bgColor: theme.palette.info.light },
+    4: { textColor: theme.palette.warning.main, bgColor: theme.palette.warning.light },
+    5: { textColor: theme.palette.warning.main, bgColor: theme.palette.warning.light },
+    6: { textColor: theme.palette.warning.main, bgColor: theme.palette.warning.light },
+  };
```

Comment:
I wonder if there's a way to do this with a default for >=4 in the future. Not concerned right now.
Edit: Nvm, I see how you handled this below.

Reply:
It's slightly clunky, and kind of assumes stable keys for the classification id. But I think it's good enough, may revisit when we wire up the api component here.